src/alfasim_sdk/_internal/alfacase/schema.py

```diff
@@ -10,9 +10,8 @@
 # for class_ in list_of_classes_that_needs_schema:
 #    cog.out(generate_alfacase_schema(class_))
 # ]]]
-from strictyaml import Any, Bool, Enum, Int, Map, MapPattern, Optional, Seq, Str, Float # noreorder
-
-
+from strictyaml import (Any, Bool, Enum, Float, Int, Map,  # noreorder
+                        MapPattern, Optional, Seq, Str)
```

Comment:
You should not directly change this.
This is generated code you must change the part inside the "cog" comment (around line 6).
The `# noreorder` is there to prevent automatic formaters to messing with this code, but maybe it is not necessary due the `# fmt: off` (at the top) already in place.